src/test/resources/uk/gov/justice/digital/hmpps/whereabouts/integration/courtAppointments.json

```diff
@@ -5,7 +5,7 @@
       "bookingId": 1,
       "appointmentId": 1,
       "court": "York",
-      "hearingType": "PRE"
+      "hearingType": "MAIN"
```

Comment:
Why did this change?

Reply:
Every video link booking must have a MAIN appointment. You can't have a solitary PRE appointment. I've move this JSON in-line becuase its the only place it is used.